streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -126,6 +126,13 @@
 
     private static final String JMX_PREFIX = "kafka.streams";
     private static final int DEFAULT_CLOSE_TIMEOUT = 0;
+    /**
+     * @TODO This only exists to allow us to pass tests in Kafka Streams
+     * We will need to fix this in a future pull request, particularly 
+     * since how much time one would need to block for Kafka Streams is 
+     * still unknown
+     */
+    private static final int DEFAULT_BLOCKING_TIME = 20000;
```

Comment:
I'm not sure I understand why this should be necessary. We're not removing any functionality from the consumer. If possible, I'd prefer to do changes for streams in a separate PR.

Reply:
Can we leave this for a follow-up? I still don't understand why we should need to touch anything in streams.

Reply:
@ConcurrencyPractitioner Can you respond here? I would prefer to keep the current streams implementation and leave improvements to a follow-up PR.

Reply:
How about retrieving StreamsConfig.REQUEST_TIMEOUT_MS_CONFIG and using that as timeout ?

Reply:
Unless there is a good reason why these changes must be here, please revert them. This will block merging the PR.

Reply:
Yeah, I agree. Streams code shouldn't show up in this PR diff at all. If there are failing streams tests, then I think there is something wrong with the KafkaConsumer implementation.